install/index.php

```diff
@@ -1847,7 +1845,7 @@ function revertSetting(defval, inpid)
 
 		if(substr($cookiedomain, 0, 4) == "www.")
 		{
-			$cookiedomain = my_substr($cookiedomain, 4);
+			$cookiedomain = substr($cookiedomain, 4);
```

Comment:
The only reason to use `my_substr` over `substr` are multi-byte UTF-8 characters, but there is none of them in the `www.` substring, only basic ASCII. So using the slower MyBB alternative here is pointless.